<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="dynamodb.UserDaoTest" tests="6" skipped="0" failures="0" errors="0" timestamp="2022-03-28T04:05:44" hostname="KrishnarthysAir" time="8.454">
  <properties/>
  <testcase name="updateUser_userDoesNotExist_throwsLoginException()" classname="dynamodb.UserDaoTest" time="6.642"/>
  <testcase name="createUser_savesUserToDatabase()" classname="dynamodb.UserDaoTest" time="0.575"/>
  <testcase name="createUser_userAlreadyExist_throwsUserAlreadyExistsException()" classname="dynamodb.UserDaoTest" time="0.377"/>
  <testcase name="getUser_userDoesNotExist_throwsLoginException()" classname="dynamodb.UserDaoTest" time="0.212"/>
  <testcase name="updateUser_userAlreadyExist_updatesIsNewUserToFalse()" classname="dynamodb.UserDaoTest" time="0.373"/>
  <testcase name="getUser_userExists_fetchesTheUser()" classname="dynamodb.UserDaoTest" time="0.263"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
